vagrant.gemspec

```
@@ -12,7 +12,7 @@ Gem::Specification.new do |s|
   s.summary       = "Build and distribute virtualized development environments."
   s.description   = "Vagrant is a tool for building and distributing virtualized development environments."
 
-  s.required_ruby_version     = "~> 2.4", "< 2.7"
+  s.required_ruby_version     = "~> 2.4"
```

I would suggest to keep the restriction on the ruby version, to prevent any breakage that could come with Ruby 2.8.
```suggestion
  s.required_ruby_version     = "~> 2.4", "< 2.8"
```

---

Speaking as a downstream maintainer, I have to say that this adds unnecessary friction to the work or testing vagrant against new ruby versions, and will force downstream to diverge. it's a little thing, but little things here and there accumulate.

besides, vagrant upstream embeds ruby in the omnibus packages, so they have full control over which ruby to include. that upper bound helps nobody.

but if the maintainers insist, by all means I can make this change.